src/popover/popover.js

```diff
@@ -512,8 +448,24 @@ class Popover extends React.Component<PopoverPropsT, PopoverPrivateStateT> {
         this.state.isMounted &&
         (this.props.isOpen || this.state.isAnimating)
       ) {
-        const mountNode = this.getMountNode();
-        rendered.push(ReactDOM.createPortal(this.renderPopover(), mountNode));
+        rendered.push(
+          <Layer
+            mountNode={this.props.mountNode}
+            onMount={() => this.setState({isLayerMounted: true})}
+            onUnmount={() => this.setState({isLayerMounted: false})}
+          >
+            <TetherBehavior
+              anchorRef={this.anchorRef.current}
+              arrowRef={this.arrowRef.current}
+              popperRef={this.popperRef.current}
+              ignoreBoundary={this.props.ignoreBoundary}
```

Comment:
following up on my comment from the other pr. a consistent `popover` interface can be maintained as well as enabling the change I suggested earlier. 

```jsx
<TetherBehavior
  popperOptions={{
    preventOverflow: {enabled: !this.props.ignoreBoundary},
    ...this.props.popperOptions,
  }}
/>
```

Reply:
Addressed it in the #1150. We can remove this prop for the popover in the next major version. I've added a comment on it here.